cypress/integration/admin/pageBuilder/category/categories.spec.js

```diff
@@ -1,3 +1,4 @@
+import { divide } from "lodash";
```

Comment:
Let's move tests that are not related to categories into a separate dedicate file, for example, `pageBuilderEditor.spec.js`.